terraI18n.config.js

```diff
@@ -0,0 +1,12 @@
+// terraI18n.config.js
+/* eslint-disable import/no-extraneous-dependencies */
+const path = require('path');
+const glob = require('glob');
+
+const searchPattern = path.resolve(process.cwd(), 'node_modules', 'terra-*', 'packages', '*', 'translations');
+
+const i18nConfig = {
+  directories: glob.sync(searchPattern),
```

Comment:
We could simplify to a regex pattern as the aggregate-translations method will glob for us
```
directories: [path.join('node_modules', 'terra-*', 'packages', '*', 'translations')],
```

Reply:
![](https://media.giphy.com/media/87y7z2LdEhxCM/giphy.gif)